tests/fixtures/optimizer/qualify_columns__with_invisible.sql

```diff
@@ -0,0 +1,68 @@
+--------------------------------------
```

Comment:
A lot of these seems copy&pasted from the normal test case.
Do we need all of them?
Duplicate tests cases can be a pain.

Reply:
I removed a lot of them. the ones I left are mainly dealing with select stars although there are a few testing direct selection of invisible columns.

some are duplicated that's true but it's still different since the schema in this test includes invisible columns. I'll look again tho to see if any don't seem worth it